### PR TITLE
Kurtwheeler fix foreman

### PR DESCRIPTION
**Pull request by kurtwheeler**
## Issue Number

This will be necessary for redeploying

## Purpose/Implementation Notes

This makes the foreman relevant again!

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

Works locally.

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: common/data_refinery_common/message_queue.py [hunk 1]
```diff
@@ -60,11 +59,13 @@ def send_job(job_type: Enum, job_id: int) -> None:
...
                 nomad_job,
                 job_type.value,
-                job_id)
+                job.id)
     try:
-        nomad_client.job.dispatch_job(nomad_job, meta={"JOB_NAME": job_type.value,
-                                                       "JOB_ID": str(job_id)})
+        nomad_response = nomad_client.job.dispatch_job(nomad_job, meta={"JOB_NAME": job_type.value,
+                                                                        "JOB_ID": str(job.id)})
+        job.nomad_job_id = nomad_response["DispatchedJobID"]
```
Comment (Miserlou) [inline common/data_refinery_common/message_queue.py]: Excellent

File: foreman/data_refinery_foreman/foreman/main.py [hunk 2]
```diff
@@ -27,9 +33,8 @@
 MAX_PROCESSOR_RUN_TIME = timedelta(days=1)
 MAX_QUEUE_TIME = timedelta(days=1)
```
Comment (Miserlou) [inline foreman/data_refinery_foreman/foreman/main.py]: Seems too high and too low, repsectively. This should mayyyybe be variablized? Although then it's more fine tuning to do.

File: common/data_refinery_common/message_queue.py [hunk 3]
```diff
@@ -18,13 +18,12 @@
 NONE_JOB_ERROR_TEMPLATE = "send_job was called with NONE job_type: {} for {} job {}"
 
 
-def send_job(job_type: Enum, job_id: int) -> None:
+def send_job(job_type: Enum, job) -> None:
```
Comment (Miserlou) [inline common/data_refinery_common/message_queue.py]: I think there are a few places elsewhere that call this function that have not had their signatures updated - transcriptome_index, API smash dispatcher,

File: common/data_refinery_common/message_queue.py [hunk 4]
```diff
@@ -60,11 +59,13 @@ def send_job(job_type: Enum, job_id: int) -> None:
...
-        nomad_client.job.dispatch_job(nomad_job, meta={"JOB_NAME": job_type.value,
-                                                       "JOB_ID": str(job_id)})
+        nomad_response = nomad_client.job.dispatch_job(nomad_job, meta={"JOB_NAME": job_type.value,
+                                                                        "JOB_ID": str(job.id)})
+        job.nomad_job_id = nomad_response["DispatchedJobID"]
+        job.save()
     except URLNotFoundNomadException:
         logger.error("Dispatching Nomad job of type %s to host %s and port %s failed.",
-                     job_type, nomad_host, nomad_port)
+                     job_type, nomad_host, nomad_port, job=str(job))
```
Comment (dongbohu) [inline common/data_refinery_common/message_queue.py]: `str(job.id)` is probably better, since job is an object now.

**Pull request merged**